### DISCUSSION TITLE
Social Icons

### DISCUSSION THREAD

**Discussion by rdoyle99** [tailwindlabs/heroicons#358] [Ideas]
Would love a set of social icons, like the Twitter icon on top of the https://heroicons.com/ page!

**Comment by djam90** [comment]
I too would love social icons. I've just been adding a few icons to a page, then got to the Facebook icon and there isn't one.

**Comment by Keshavdulal** [comment]
Having to manage Social Icons (FB/TT/IG/LI/YT etc) separately is a bit of hassle.

**Comment by zefir-git** [comment]
IMO brand icons have **NO** place in Heroicons. Just use [simpleicons.org](https://simpleicons.org/) which is specifically for brand icons
